src/components/Navigation/NavBar.scss

```diff
@@ -122,9 +122,6 @@ $widget-height: 50px;
 }
 
 .spacing {
-  width: 100%;
-  height: $logo-height - $widget-height;
-
```

Comment:
Ikke? Følg med på menylinjen når du har bred skjerm.

Her er to bilder som viser forskjellen. Uten linjene:
<img width="1140" alt="uten-linjene" src="https://user-images.githubusercontent.com/8851134/28379953-dbe35c92-6cb5-11e7-91d0-f09229f56e7b.png">

Med linjene:
<img width="1146" alt="med-linjene" src="https://user-images.githubusercontent.com/8851134/28379965-e45a75e0-6cb5-11e7-89a4-3bb1b2a3e87e.png">